cargo-apk/README.md

```diff
@@ -192,6 +192,12 @@ port = "8080"
 path = "/rust-windowing/android-ndk-rs/tree/master/cargo-apk"
 path_prefix = "/rust-windowing/"
 mime_type = "image/jpeg"
+
+# Set up reverse port forwarding through `adb reverse`, meaning that if the
+# Android device connects to `https://localhost:1338` it will be routed to 
+# the localhost on the host device instead.
```

Comment:
This isn't strictly for `http`, I'd write ``Android device connects to `localhost` on port `1338` ...``.

For demonstration purposes, can we use a different source and destination purposes? e.g. ``it will be routed to the host on port `XYZ whatever you choose` instead.``